savant/config/module_config.py

```diff
@@ -152,34 +173,127 @@ def setup_batch_size(config: Module) -> None:
             'for parameter "batch_size". Allowed values: 1 - 1024.'
         )
     config.parameters['batch_size'] = batch_size
+    logger.info('Pipeline batch size is set to %s.', batch_size)
 
 
-def resolve_parameters(config: DictConfig):
+def configure_module_parameters(module_cfg: DictConfig) -> None:
     """Resolve parameters on module config ("frame", "draw_func", etc.).
 
     :param config: Module config.
     """
+    if 'parameters' not in module_cfg or module_cfg.parameters is None:
+        module_cfg.parameters = {}
+        return
+
+    def apply_schema(cfg: dict, node: str, schema_class: Any) -> None:
+        schema = OmegaConf.structured(schema_class)
+        if node not in cfg or cfg[node] is None:
```

Comment:
`draw_func` is optional. When it's `None` or absent it should stay that way. I suggest to add `default` parameter to this method.

Here `draw_func` is always enabled.

Reply:
Fixed.